src/components/tasks/card/index.tsx

```diff
@@ -30,9 +31,7 @@ const Card: FC<Props> = ({
   const statusRedList = [BLOCKED];
   const statusNotOverDueList = [COMPLETED, VERIFIED, AVAILABLE];
   const cardDetails = content;
-  const [assigneeProfilePic, setAssigneeProfilePic] = useState(
-    `${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}/${cardDetails.assignee}/img.png`,
-  );
+  const [assigneeProfilePic, setAssigneeProfilePic] = useState<string>('');
```

Comment:
[nitpick] Do we need to specify the type of state value since the default value would anyways imply to string?

Reply:
Yes you are right there is  no need to specify that. I will remove it